locales/en-US/strings.properties

```diff
@@ -36,6 +36,8 @@ menu_action_open_new_window=Open in a New Window
 menu_action_open_private_window=Open in a New Private Window
 menu_action_dismiss=Dismiss
 menu_action_delete=Delete from History
+confirm_history_delete=Are you sure you want to delete every instance of this page from your history?
```

Comment:
Maybe you can leave this as-is with the following changes:

- Rename them to `confirm_history_delete_notice_p1` and `confirm_history_delete_notice_p2`
- Add a comment `# LOCALIZATION NOTE: These two strings appear together in the same dialog `  or something along those lines